app/controllers/admin/groups_controller.rb

```diff
@@ -58,4 +61,12 @@ def group_params
       group_memberships_attributes: [:id, :person_id, :_destroy]
     )
   end
+
+  def membership_person_id_params
+    if group_params[:group_memberships_attributes]
+      group_params[:group_memberships_attributes].values.reject { |attributes| attributes[:_destroy] == "1" || attributes[:person_id].empty? }.map { |attributes| attributes[:person_id] }
```

Comment:
This is a little opaque. I think we should consider refactoring this a little here by splitting it into stages, or at least multilining it.

```ruby
group_params[:group_memberships_attributes].values.reject { |attributes|
  attributes[:_destroy] == "1" || attributes[:person_id].empty?
}.map { |attributes| attributes[:person_id] }
```

I also think we might be able to simplify this with a reduce, as here we're iterating over the collection twice.

Consider:

```ruby
group_params[:group_memberships_attributes].values.reduce([]) do |result, attributes|
  next if (attributes[:_destroy] == "1" || attributes[:person_id].empty?)
  result << attributes[:person_id]
end
```

This combines the reject and the map into a single statement, traversing the params only once.